RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -783,6 +794,8 @@ edm::ParameterSetDescription EcalUncalibRecHitWorkerMultiFit::getAlgoDescription
               edm::ParameterDescription<double>("outOfTimeThresholdGain61mEE", 1000, true) and
               edm::ParameterDescription<double>("amplitudeThresholdEB", 10, true) and
               edm::ParameterDescription<double>("amplitudeThresholdEE", 10, true) and
+              edm::ParameterDescription<bool>("useSlewCorrectionEB", true, true) and
```

Comment:
Maybe change the variable name to `crossCorrelationUseSlewCorrectionEB` to distinguish it clearly from the slew rate fixes in the amplitude reconstruction, which are activated with the `gainSwitchUseMaxSampleEB` parameter.